indigo_social/migrations/0005_import_google_profile_pictures.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-04-15 08:46
+from __future__ import unicode_literals
+
+from django.core.files import File
+from django.db import migrations
+
+from urllib.request import urlretrieve, urlcleanup
+
+
+def import_google_profile_pictures(apps, schema_editor):
+	SocialAccount = apps.get_model('allauth.socialaccount', 'SocialAccount')
+	UserProfile = apps.get_model('indigo_social', 'UserProfile')
+
+	def fetch_and_save_profile_photo(user_profile, url):
+		try:
```

Comment:
Pls change tabs to spaces.

The for loop on line 22 is over-indented and so won't run.